### PR TITLE
Update uralamin.json

### PR DESCRIPTION
**Pull request by MrAlaminH**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://retro-folio.vercel.app/

### CODE REVIEW

**Comment by EducatedSuddenBucket** [comment]
AAAA Records should be a array

File: domains/uralamin.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
         "email": "alaminiam110@gmail.com"
     },
     "record": {
-        "CNAME": "uralamin.onrender.com"
+        "AAAA": ["2600:1901:0:6d85::"]
```
Comment (Stef-00012) [inline domains/uralamin.json:7]: missing a comma
```suggestion
        "AAAA": ["2600:1901:0:6d85::"],
```

File: domains/uralamin.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
...
+        "A": ["34.120.54.55"]
```
Comment (Stef-00012) [inline domains/uralamin.json:8]: Can you show a preview of your website?

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/uralamin.json (hunk 2 above)
Comment (MrAlaminH) [reply domains/uralamin.json]: preview from Vercel or where?

**Comment by MrAlaminH** [comment]
Register _acme-challenge.uralamin with CNAME record pointing to 1b53ef83ff99303ae27f8be2._acme.deno.dev.

**Pull request closed without merging**